osquery/filesystem/fileops.h

```diff
@@ -16,7 +16,15 @@
 
 #ifdef WIN32
 
+#include "accctrl.h"
```

Comment:
Considering most of the logic you originally had implemented has moved to the fileops files, are _all_ of these header includes necessary still? Would you mind moving the headers you need to the `fileops.cpp` impl file instead?

Reply:
Good call. I removed a few.